account-management/src/test/java/com/aprilhorizon/accountmanagement/AccountManagementApplicationTests.java

```diff
@@ -1,13 +1,22 @@
 package com.aprilhorizon.accountmanagement;
 
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
 @SpringBootTest
 class AccountManagementApplicationTests {
```

Comment:
Integration tests are written for controller classes so we can name the class `AccountControllerTests`. I have created a class with that name and added empty test cases which you can fill in.